### ISSUE TITLE
Add istio 1.10+ support for glooctl istio inject

### ISSUE THREAD

**Issue by kevin-shelaga** [solo-io/gloo#5224]
**Is your feature request related to a problem? Please describe.**
Add istio 1.10+ support for glooctl istio inject

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Sodman** [comment]
Whoever picks this up please talk to @christian-posta or myself, I believe there's a better/more generic way to do this using istio injection rather than defining a new container for each istio release, which will be more maintainable going forward.

CC @sam-heilbron

**Comment by sam-heilbron** [comment]
> Whoever picks this up please talk to @christian-posta or myself, I believe there's a better/more generic way to do this using istio injection rather than defining a new container for each istio release, which will be more maintainable going forward.
> 
> CC @sam-heilbron

Agreed, good call. I'll write up a larger (broader?) issue to support istio inject generically so we don't keep chasing our tail.

**Comment by kevin-shelaga** [comment]
@sam-heilbron should we just implement 1.11 the "old way" for now with this ticket?
